lib/Event/Admin/Login/LoginRedirectEvent.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace Pimcore\Event\Admin\Login;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class LoginRedirectEvent extends Event
+{
+
+    /**
+     * @var string
+     */
+    protected $routeName;
+
+    /**
+     * @var array
+     */
+    protected $routeParams;
+
+    /**
+     * @param string $routeName
+     * @param array $routeParams
+     */
+    public function __construct(string $routeName, array $routeParams)
```

Comment:
```suggestion
    public function __construct(string $routeName, array $routeParams = [])
```
I think this could be optional, right?